### PR TITLE
feat: add foundations app to test colors, icons and fonts

### LINKED ISSUE #1405: Add another vite app in foundations for fonts, colors & icons
We should add another vite app in foundations with this file and maybe colors & icons. So we could watch it separately from components?

_Originally posted by @nmerget in https://github.com/db-ui/mono/pull/1373#discussion_r1292998780_

### PR DESCRIPTION
**Pull request by nmerget**
## Proposed changes

closes #1405 

## Types of changes

-   [ ] Bugfix (non-breaking change which fixes an issue)
-   [x] New feature (non-breaking change which adds functionality)
-   [ ] Refactoring (fix on existing components or architectural decisions)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-   [ ] Documentation Update (if none of the other choices apply)

### CODE REVIEW

File: showcases/angular-showcase/src/app/components/form/form.component.html [hunk 1]
```diff
@@ -133,6 +133,7 @@ <h6>Reactive Forms Example:</h6>
 				</ul>
 				<p>Checkbox:</p>
 				<db-checkbox
+					id="bla"
```
Comment (mfranzke) [inline showcases/angular-showcase/src/app/components/form/form.component.html:136]:
```suggestion
					id="bahn"
```

File: .github/workflows/02-e2e-foundations.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
+name: 🎭 Playwright E2E
+
+on:
+  workflow_call:
```
Comment (mfranzke) [inline .github/workflows/02-e2e-foundations.yml:4]: I'm not that familiar with the configuration syntax, but isn't a value missing here?

Comment (nmerget) [reply .github/workflows/02-e2e-foundations.yml:4]: It's fine we don't have something like `input` or `output` values for this workflow

**Comment by mfranzke** [comment]
@nmerget there are merge conflicts that came up in the meantime.

File: packages/foundations/scss/_default.assets-paths.scss
Comment (mfranzke) [inline packages/foundations/scss/_default.assets-paths.scss:1]: @nmerget why is a change like this necessary? We couldn't expect the `assets` folder to be located in `root` for any environment that consumes our files, but having those references included in relation to the CSS position might be more universal.

File: packages/components/package.json
Comment (mfranzke) [inline packages/components/package.json:1]: @nmerget as you include `cssnano` in the workspaces, do we still need it as a dependency in root level `package.json` than?

File: packages/foundations/test/playwright.config.mjs [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+		},
+
+		{
+			name: 'webkit',
+			use: {
+				...devices['Desktop Safari']
+			}
+		}
+	],
+
```
[the remark this answers is not in the document]
Comment (nmerget) [reply packages/foundations/test/playwright.config.mjs:63]: The branch is old, there wasn't the playwright update with the `:has` firefox

[the remark this answers is not in the document]
Comment (mfranzke) [reply packages/foundations/test/playwright.config.mjs]: I've just added it now.

File: packages/foundations/scss/_default.assets-paths.scss
Comment (nmerget) [reply packages/foundations/scss/_default.assets-paths.scss:1]: fixed

File: packages/foundations/scss/db-ui-foundations-weboack.scss
Comment (mfranzke) [inline packages/foundations/scss/db-ui-foundations-weboack.scss:1]: rename `packages/foundations/scss/db-ui-foundations-weboack.scss` to `packages/foundations/scss/db-ui-foundations-webpack.scss`

File: packages/foundations/scss/icons/include-relative.scss
Comment (mfranzke) [inline packages/foundations/scss/icons/include-relative.scss:1]: the resulting paths `./assets/*/` in this file don't work in this subfolder. Let's talk.

File: packages/foundations/scss/icons/include.scss [hunk 4]
```diff
@@ -1,6 +1,13 @@
 @use "icon-helpers";
 @use "../default.assets-paths" as *;
 
+%icon-alignment {
+	&:is(span) {
+		display: inline-flex;
+		vertical-align: baseline;
+	}
+}
+
```
Comment (mfranzke) [inline packages/foundations/scss/icons/include.scss:10]:
```suggestion
[data-icon],
[data-icon-before],
[data-icon-after] {
	&:is(span) {
		display: inline-flex;
		vertical-align: baseline;
	}
}

```

You could instead write it like this and than you wouldn't need to `SCSS placeholder` and references within the other selectors.

File: packages/foundations/scss/icons/include-relative.scss
Comment (nmerget) [reply packages/foundations/scss/icons/include-relative.scss:1]: Which subfolder?

Comment (nmerget) [reply packages/foundations/scss/icons/include-relative.scss:1]: removed it, like discussed

**Review by mfranzke — APPROVED** [review]

**Pull request merged**